eclipsedataspaceconnector/api-wrapper/.gitignore

```
@@ -4,3 +4,4 @@ build/
 adapter/
 semantic-hub/
 registry/
+!*.pfx
```

Please add an empty line at the end

```suggestion
!*.pfx

```